osu.Game.Rulesets.Osu/UI/Cursor/CursorTrail.cs

```diff
@@ -54,7 +54,8 @@ public CursorTrail()
 
             for (int i = 0; i < max_sprites; i++)
             {
-                parts[i].InvalidationID = 0;
+                // InvalidationID 1 forces an update of each part of the cursor trail the first time ApplyState is ran on the draw node
```

Comment:
```suggestion
                // InvalidationID 1 forces an update of each part of the cursor trail the first time ApplyState is run on the draw node
```

can you mention that this is required to avoid (outdated|garbled|corrupted) data being displayed (choose the word that describes what is actually happening – coming into this i'm not sure yet).